src/main/java/uk/gov/hmcts/reform/roleassignmentrefresh/domain/service/process/RefreshJobsOrchestrator.java

```diff
@@ -109,11 +147,141 @@ private void sendJobToORMService(Long jobId, UserRequest userRequest) {
         }
     }
 
-    public void triggerRASUserCount() {
-        ResponseEntity<Object> responseEntity =  userCountService.getRasUserCounts();
+    public ResponseEntity<CountResponse> triggerRASUserCount() {
+        ResponseEntity<CountResponse> responseEntity =  userCountService.getRasUserCounts();
         if (responseEntity.getStatusCode() != HttpStatus.OK) {
             log.error("Error return from RAS User Count: " + responseEntity.toString());
         }
+        return responseEntity;
+    }
+
+
+    public List<Count> compareCounts(CountResponse before, CountResponse after, String countName) {
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.configure(JsonParser.Feature.ALLOW_UNQUOTED_FIELD_NAMES, true);
```

Comment:
Can you remove the mapper if it is no longer used here.

Reply:
removed

Reply:
Resolved in a3d32427ca1ed13333909504af43ca392313e622.